### PR TITLE
chore: ignore .build folder for swift

### PR DESCRIPTION
**Pull request by DoronElir**
- [ ] Follows [CONTRIBUTING](https://github.com/snyk/snyk/blob/master/CONTRIBUTING.md) rules

### CODE REVIEW

File: src/lib/find-files.ts [hunk 1]
```diff
@@ -62,14 +66,18 @@ export async function find(
   const found: string[] = [];
   const foundAll: string[] = [];
 
-  // ensure we ignore find against node_modules path.
-  if (path.endsWith('node_modules')) {
+  // ensure we ignore find against node_modules path and .build folder for swift.
+  if (path.endsWith('node_modules') || path.endsWith('.build')) {
```
Comment (orsagie) [inline src/lib/find-files.ts:70]: Maybe what we need to do here is to parse the path and look that the `.build` is not part of something like yyy.xxx.build?

File: test/tap/find-files.test.ts [hunk 2]
```diff
@@ -108,6 +108,19 @@ test('find package.json file in test fixture ignoring node_modules', async (t) =
   t.same(result.sort(), expected.sort(), 'should return expected file');
 });
 
+test('find Package.swift file in test fixture ignoring .build folder', async (t) => {
```
Comment (orsagie) [inline test/tap/find-files.test.ts:111]: Make a test for the case of a/b/c.build

**Review by orsagie — APPROVED** [review]
One comment, but LGTM

**Comment by Greg-Smulko** [comment]
This change broke our workflow! 😡 😉 https://xkcd.com/1172/

We have `paket.dependencies` and `paket.lock` files in a `.build` folder, together with other build-related stuff. As a result of this change, these paket files are ignored by Snyk. 😞

**Pull request merged**